tests/ga/test_extension.py

```diff
@@ -3431,17 +3431,19 @@ def mock_http_put(url, *args, **_):
                         }
                     }
                 },
-                "agentName": AGENT_NAME,
-                "daemonVersion": "0.0.0.0",
-                "pythonVersion": "Python: {0}.{1}.{2}".format(PY_VERSION_MAJOR, PY_VERSION_MINOR, PY_VERSION_MICRO),
-                "extensionSupportedFeatures": [name for name, _ in
-                                             get_agent_supported_features_list_for_extensions().items()],
-                "crpSupportedFeatures": [
+                "supportedFeatures": [
```

Comment:
This test will fail whenever a new supported feature is added to this list. Might be worthwhile to make this list dynamic too? (Something similar to how you're doing it for extensionSupportedFeatures list below)

Reply:
Good point. Updated the logic.